<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2400</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SAMLabel(by:Eric)</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QScrollArea" name="leftAreaImg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="right">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>983</width>
             <height>1284</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="rightAreaImg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="left">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>983</width>
             <height>1284</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>2</number>
              </property>
              <property name="movable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="SAM">
               <attribute name="title">
                <string>SAM</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <layout class="QVBoxLayout" name="SAMLayout"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="segView">
               <attribute name="title">
                <string>segView</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <layout class="QVBoxLayout" name="segViewLayout"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="promptView">
               <attribute name="title">
                <string>promptView</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="promptViewLayout"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="oriView">
               <attribute name="title">
                <string>oriView</string>
               </attribute>
               <layout class="QVBoxLayout" name="oriView_2">
                <item>
                 <layout class="QVBoxLayout" name="oriViewLayout"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>394</width>
         <height>1288</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>361</width>
             <height>61</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>setting</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>分割细粒度</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>确认设置</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_4">
          <property name="styleSheet">
           <string notr="true">QScrollArea{
	background-color: rgb(255, 255, 255 ,150);
}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>374</width>
             <height>298</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_6">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_6">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>354</width>
                 <height>260</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="label_layout"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_5">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>374</width>
             <height>897</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,100">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="ImageList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_7">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>354</width>
                 <height>859</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="ImageLayout"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actiongithub"/>
   </widget>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openDoc"/>
   <addaction name="lastFile"/>
   <addaction name="nextFile"/>
  </widget>
  <action name="openDoc">
   <property name="icon">
    <iconset>
     <normaloff>ico/opendoc.png</normaloff>ico/opendoc.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="lastFile">
   <property name="icon">
    <iconset>
     <normaloff>ico/last.png</normaloff>ico/last.png</iconset>
   </property>
   <property name="text">
    <string>上一个文件</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="nextFile">
   <property name="icon">
    <iconset>
     <normaloff>ico/nest.png</normaloff>ico/nest.png</iconset>
   </property>
   <property name="text">
    <string>下一个文件</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actiongithub">
   <property name="text">
    <string>github</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico/ico.qrc"/>
 </resources>
 <connections/>
</ui>
